iocore/net/I_NetVConnection.h

```diff
@@ -179,6 +179,10 @@ struct NetVCOptions {
 
   EventType etype;
 
+  /** ALPN protocol-lists. The format is OpenSSL protocol-lists format (vector of 8-bit length-prefixed, byte strings)
+      https://www.openssl.org/docs/manmaster/man3/SSL_CTX_set_alpn_protos.html
+   */
+  std::string_view alpn_protos;
```

Comment:
Can we get length from `ats_scoped_str ` easily? Main motivation of using `std::string_view` here is that.